en/concept/mixing.md

```diff
@@ -283,6 +283,10 @@ S: <group> <index> <-ve scale> <+ve scale> <offset> <lower limit> <upper limit>
 The `S:` lines must be below the `O:` line.
 :::
 
+:::note
```

Comment:
Thanks @bkueng! I've re-written this slightly. Still good.

Reply:
@bkueng I've merged this because I think the modifications I made look fine. But happy to take late comment if you think I broke it.

Reply:
yup